core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/store/ai-content-prompt.store.ts

```diff
@@ -13,28 +13,49 @@ export interface AiContentPromptState {
     content: string;
     open: boolean;
     acceptContent: boolean;
+    deleteContent: boolean;
+    exit: boolean;
 }
 
 @Injectable({
     providedIn: 'root'
 })
 export class AiContentPromptStore extends ComponentStore<AiContentPromptState> {
     constructor(private dotAiService: DotAiService) {
-        super({ prompt: '', loading: false, content: '', open: false, acceptContent: false });
+        super({
+            prompt: '',
+            loading: false,
+            content: '',
+            open: false,
+            acceptContent: false,
+            deleteContent: false,
+            exit: false
```

Comment:
loading, open and exit... is there any way more that one can happen at the same time? maybe we do like: `state: 'idle' | 'loading' | 'open'`?

Reply:
agree moved to `status`